samples/drivers/display/src/main.c

```diff
@@ -296,6 +296,7 @@ int main(void)
 	buf_desc.pitch = capabilities.x_resolution;
 	buf_desc.width = capabilities.x_resolution;
 	buf_desc.height = h_step;
+	buf_desc.frame_incomplete = false;
```

Comment:
We should be setting `frame_incomplete` to true when we write the last buffer before we call `display_blanking_off`, right? I'm just thinking this would be useful to demonstrate the usage of this field

Reply:
I'm unsure what you mean, isn't that frame already complete?

Reply:
Sorry, my bad! Didn't realize there are multiple writes. Yes, weil change that.

Reply:
Fixed